systemtest/src/main/java/io/strimzi/systemtest/utils/specific/KeycloakUtils.java

```diff
@@ -53,6 +56,30 @@ public static void deleteKeycloak(final String deploymentNamespace, final String
         Exec.exec(Level.INFO, "/bin/bash", PATH_TO_KEYCLOAK_TEARDOWN_SCRIPT, deploymentNamespace, getValidKeycloakVersion(), watchNamespace);
     }
 
+    public static void deployPostgres(final String deploymentNamespace) {
+        deployPostgres(deploymentNamespace, "main");
+    }
+
+    public static void deployPostgres(final String deploymentNamespace, final String branch) {
+        LOGGER.info("Deploy Postgres DB and its Operator Operator branch:{} in namespace: {}", branch, deploymentNamespace);
+
+        TestUtils.waitFor("Keycloak instance readiness", Constants.KEYCLOAK_DEPLOYMENT_POLL, Constants.KEYCLOAK_DEPLOYMENT_TIMEOUT, () -> {
+            ExecResult result = Exec.exec(Level.INFO, "/bin/bash", PATH_TO_PQO_PREPARE_SCRIPT, deploymentNamespace, branch);
+
+            if (!result.out().contains("Postgres Operator and Database successfully deployed")) {
+                LOGGER.info("Errors occurred during Keycloak install: {}", result.err());
+                return false;
+            }
+            return  true;
+        });
+        LOGGER.info("Postgres DB (Operator) in namespace {} is ready", deploymentNamespace);
+    }
+
+    public static void deletePostgres(final String deploymentNamespace) {
+        LOGGER.info("Teardown Postgres DB and its Operator in namespace: {}", deploymentNamespace);
+        Exec.exec(Level.INFO, "/bin/bash", PATH_TO_PQO_TEARDOWN_SCRIPT, deploymentNamespace);
```

Comment:
So you think that it will be deleted without any issue? You don't need wait or something?

Reply:
It's nice that you think it will be deleted right after your script, but there should be some wait for deletion, to prevent any race conditions that might happen.

Reply:
Valid point, I am checking this one a bit more as teardown script should have such checks, despite it's working fine.
I will try to figure out something with it

Reply:
Actually, `kubectl delete` has `wait`, which should do the trick for us ;)
```      
--wait=true: If true, wait for resources to be gone before returning. This waits for finalizers.
```